randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -560,7 +560,7 @@ Extra - room_id: [18]
   * L0 Hatch to Zazabi Speedway/Door to Zazabi Arena
   * Extra - door_idx: (133,)
   > Door to Zazabi Access
-      Impossible
+      Screw Attack and Speed Booster and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
You're missing a `After Zazabi`.
And that makes me wonder, whether this is even necessary in the first place. Even if you assume starting location in speedway with the necessary items, you're forced to kill zazabi first. and from there you cant save or anything so you need HJ/SJ to get up.